utils/create-preview-links.ts

```diff
@@ -23,30 +23,27 @@ export const getQuickstartsFromPRFiles = async (
   token: string
 ): Promise<string[]> => {
   const filesURL = `${prURL}/files`;
-
   const files = await fetchPaginatedGHResults(filesURL, token);
-
-  const quickstartPaths = filterOutTestFiles(files)
+  const nonTestFiles = filterOutTestFiles(files);
+  
+  const quickstartPaths = nonTestFiles
+      .map(prop('filename'))
+      .filter(filePath => QUICKSTART_CONFIG_REGEXP.test(filePath))
+      .map(filePath => path.dirname(filePath.split('quickstarts/').pop()!))
+
+  /*
+    We can rely on a quickstart existing, and use the quickstart class with 
+    getRelatedQuickstarts to get all quickstarts related to the component.  If
+    the component itself exists we know a quickstart related to it also exists.
```

Comment:
```suggestion
    We can rely on a quickstart existing on the `main` branch because if it isn't, then it was created in this 
    pull request and will be caught by the logic above. We can then use the Quickstart class with 
    getRelatedQuickstarts to get all quickstarts related to the component.  If
    the component itself exists we know a quickstart related to it also exists.
```